CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ add_subdirectory(examples)
 ################################################################################
 install(
   DIRECTORY hpxio
-  DESTINATION include/hpxio
+  DESTINATION include
```

Comment:
That is exactly the case, all of the header files are already in hpxio sub-directory. Installing it directly into include/ copies the hpxio sub-directory into include directory.
In previous version we had .../include/hpxio/hpxio/foo.hpp which I thought it shouldn't be correct.